src/annotation-context/annotation/annotation-popover.tsx

```diff
@@ -78,16 +77,9 @@ export function AnnotationPopover({
   onPreviousButtonClick,
   i18nStrings,
 }: AnnotationPopoverProps) {
-  const dismissButtonRefCallback = useCallback(
-    (element: ButtonProps.Ref) => {
-      if (element) {
-        element.focus({ preventScroll: true });
-        // Falls back to alignTop on IE11
-        scrollElementIntoView(trackRef.current ?? undefined);
-      }
-    },
-    [trackRef]
-  );
+  useEffect(() => {
+    scrollElementIntoView(trackRef.current ?? undefined);
```

Comment:
When this is fired?

From the code it looks like the scroll is made only once - when the annotation popover is rendered (but not when it is opened or reopened)?

Reply:
Oh, I see - thanks. I think it would make sense to explain this in the comments above the use-effect.